en/chromebook_setup/instructions.md

```diff
@@ -1,187 +1,35 @@
-            'filename': 'debug.log',
-        },
-    },
-    'loggers': {
-        'django': {
-            'handlers': ['file'],
-            'level': 'DEBUG',
-            'propagate': True,
-        },
-    },
-}
-```
-This will create a `debug.log` file detailing Django operations and any error messages that might come up, making it much easier to fix if your website does not work.
-
-The initial restarting of the Glitch project should fail.
-(If you click on the top dropdown button `Show` then click on `In a New Window`, you will receive a `DisallowedHost` error message.)
-Do not worry about it at this stage, the tutorial will fix this as soon as you update the Django settings of your project in the `mysite/settings.py` file.
-
-### Virtual Environment
-
-A virtual environment (also called a virtualenv) is like a private box we can
-stuff useful computer code into for a project we're working on. We use them to
-keep the various bits of code we want for our various projects separate so
-things don't get mixed up between projects.
-
-Run:
-
-{% filename %}Cloud 9{% endfilename %}
-```
-mkdir djangogirls
-cd djangogirls
-python3 -m venv myvenv
-source myvenv/bin/activate
-pip install django~={{ book.django_version }}
-```
-
-(note that on the last line we use a tilde followed by an equal sign: `~=`).
-
-### GitHub
-
-Make a [GitHub](https://github.com) account.
-
-### PythonAnywhere
-
-The Django Girls tutorial includes a section on what is called Deployment,
-which is the process of taking the code that powers your new web application
-and moving it to a publicly accessible computer (called a server) so other
-people can see your work.
-
-This part is a little odd when doing the tutorial on a Chromebook since we're
-already using a computer that is on the Internet (as opposed to, say, a laptop).
-However, it's still useful, as we can think of our Cloud 9 workspace as a place
-for our "in progress" work and Python Anywhere as a place to show off our stuff
-as it becomes more complete.
-
-Thus, sign up for a new Python Anywhere account at
-[www.pythonanywhere.com](https://www.pythonanywhere.com).
+For people using a Chromebook or those with limited memory on their laptops, we recommend using a cloud IDE environment. This allows you to interact with the command line, Python, and Django directly through your browser, where a code editor is already installed.
+
+Your installation experience will be a little different.
+
+- Make a [GitHub](https://github.com) account.
+- Create a GitHub project. Visit [this](https://github.com/new) link, choose any name for your project, and make it public (needed for deployment, you can make it private later). Also, add a `README.md` and `.gitignore` file.
+
+![](../chromebook_setup/images/create-new-repo.png)
+
+In case you using this instruction, some steps of the tutorial don't have to be done again ("Deploy/Git" [part](https://tutorial.djangogirls.org/en/deploy/#installing-git)). 
```

Comment:
In case the list is kept, should this be part of the list? (Currently it splits it into separate lists.) If you want to make it part of a previous list item, indent it in the markdown source.

Reply:
I've made it a part of the previous list